sharedTest/src/main/kotlin/com/sharedtest/persistence/remote/FakeRemoteDataStore.kt

```diff
@@ -35,7 +36,7 @@ import javax.inject.Singleton
 @Singleton
 class FakeRemoteDataStore @Inject internal constructor() : RemoteDataStore {
   var lois = emptyList<LocationOfInterest>()
-  var surveys = emptyList<Survey>()
+  var surveys = listOf<Survey>(SURVEY)
```

Comment:
Instead of changing the default for all tests, let's update it for the specific tests like this:

https://github.com/google/ground-android/blob/a70cd529ca817c3a1c713e85b5ea19d7027eba47/ground/src/test/java/com/google/android/ground/repository/SurveyRepositoryTest.kt#L70